plugin/trino-influxdb/src/test/java/io/trino/plugin/influxdb/TestInfluxConnectorTest.java

```diff
@@ -0,0 +1,151 @@
+            default -> super.hasBehavior(connectorBehavior);
+        };
+    }
+
+    @Test
+    @Override
+    public void testShowCreateTable()
+    {
+        String catalog = getSession().getCatalog().orElseThrow();
+        String schema = getSession().getSchema().orElseThrow();
+        assertThat(computeScalar("SHOW CREATE TABLE orders"))
+                // If the connector reports additional column properties, the expected value needs to be adjusted in the test subclass
+                .isEqualTo(format("""
+                                CREATE TABLE %s.%s.orders (
+                                   clerk varchar,
+                                   comment varchar,
+                                   custkey bigint,
+                                   orderdate varchar,
+                                   orderkey bigint,
+                                   orderpriority varchar,
+                                   orderstatus varchar,
+                                   shippriority bigint,
+                                   totalprice double
+                                )""",
+                        catalog, schema));
+    }
+
+    @Test
+    @Override
+    public void testShowColumns()
+    {
+        MaterializedResult actual = computeActual("SHOW COLUMNS FROM orders");
+        MaterializedResult expected = resultBuilder(getSession(), VARCHAR, VARCHAR, VARCHAR, VARCHAR)
+                .row("clerk", "varchar", "", "")
+                .row("comment", "varchar", "", "")
+                .row("custkey", "bigint", "", "")
+                .row("orderdate", "varchar", "", "")
+                .row("orderkey", "bigint", "", "")
+                .row("orderpriority", "varchar", "", "")
+                .row("orderstatus", "varchar", "", "")
+                .row("shippriority", "bigint", "", "")
+                .row("totalprice", "double", "", "")
+                .build();
+
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    @Override
+    public void testDescribeTable()
+    {
+        MaterializedResult expectedColumns = MaterializedResult.resultBuilder(getSession(), VARCHAR, VARCHAR, VARCHAR, VARCHAR)
+                .row("clerk", "varchar", "", "")
+                .row("comment", "varchar", "", "")
+                .row("custkey", "bigint", "", "")
+                .row("orderdate", "varchar", "", "")
+                .row("orderkey", "bigint", "", "")
+                .row("orderpriority", "varchar", "", "")
+                .row("orderstatus", "varchar", "", "")
+                .row("shippriority", "bigint", "", "")
+                .row("totalprice", "double", "", "")
+                .build();
+        MaterializedResult actualColumns = computeActual("DESCRIBE orders");
+        assertEquals(actualColumns, expectedColumns);
+    }
+
+    @Test
+    @Override
+    public void testSelectAll()
+    {
+        // List columns explicitly, as there's no defined order in InfluxDB
```

Comment:
Is it possible to support deterministic column orders using table comment or something likes Cassandra connector?

Reply:
I carefully checked the docs and found it difficult, because there is no `comment` or some field else for us to mark orders. In InfluxDB, a point represents a single data record, similar to a row in a SQL database table,  but it only has a tag set, a field key, a field value, and a timestamp..    : (

Reply:
Is there an existing issue in InfluxDB? https://github.com/influxdata/influxdb/issues/15957? Could you please add a link to the comment?

Reply:
ok